py4cast/datasets/compute_dataset_stats.py

```diff
@@ -20,12 +21,13 @@ def compute_mean_std_min_max(
     sum_squares = torch.zeros(n_features)
     ndim_features = len(named_tensor.tensor.shape) - 1
     flat_input = named_tensor.tensor.flatten(0, ndim_features - 1)  # (X, Features)
-    best_min = torch.min(flat_input, dim=0).values
-    best_max = torch.max(flat_input, dim=0).values
+    best_min = torch.min(torch.nan_to_num(flat_input, torch.inf), dim=0).values
+    best_max = torch.max(torch.nan_to_num(flat_input, -torch.inf), dim=0).values
 
-    if torch.isnan(best_min).any() or torch.isnan(best_max).any():
-        raise ValueError(
-            "Your dataset contain NaN values, which prevent the calculation of statistics."
+    if torch.isnan(flat_input).any():
+        flat_input = torch.nan_to_num(flat_input)
```

Comment:
Explicit replace value.
```suggestion
        flat_input = torch.nan_to_num(flat_input, nan=0)
```